tests/unit/test_jobs.py

```diff
@@ -59,10 +58,10 @@
                 JobConfig(
                     type=JobType.copr_build,
                     trigger=JobConfigTriggerType.pull_request,
-                    metadata={},
```

Comment:
Whoops, sorry for not changing this in #548.

You don't have to specify the `metadata` argument at all, cause it will be [set to JobMetadataConfig()](https://github.com/packit-service/packit/blob/master/packit/config/job_config.py#L122) anyway.

Reply:
No problem. It was not used. I'll remove the empty ones.

I am just wondering why mypy does not tell us about that. (PyCharm shows that problem.)